docs/modules/ROOT/pages/how-tos/migrate-to-lokistack.adoc

```diff
@@ -0,0 +1,96 @@
+= Migrate to Lokistack
+
+Instructions to migrate from Elasticsearch Logging to Lokistack.
+
+Migrating the logs isn't part of this How-To, instead you can keep Elasticsearch logging until the retention time is expired. Then it's safe to remove Elasticsearch Logging.
+
+[TIP]
+====
+Running Lokistack and Elasticsearch Logging at the same time might exceed the resources of the infra nodes. Check in advance that you have enough resources available.
+====
+
+
+== Keep Elasticsearch
+
+To keep Elasticseach Logging just make sure the component is enabled:
+[source,yaml]
+----
+components:
+  elasticsearch:
+    enabled: true
+----
+
+
+== Enable Lokistack and make it default
+
+Enable the Lokistack component:
```

Comment:
I'd add something along the lines of "implicitly makes it the default logging stack"